src/index.js

```diff
@@ -191,9 +200,15 @@ function possibleMoves(squares, player, i, j) {
 function getPossibleMovesInDirection(squares, direction, i, j, player) {
   const oppPlayer = player === 'X' ? 'O' : 'X';
   const directionMoves = [];
+  if (getDirectionIndex(direction, i, j) == null ){
+    return [];
+  }
   [i, j] = getDirectionIndex(direction, i, j);
   while (squares[i][j] === oppPlayer && i !== null && j !== null) {
     directionMoves.push([i,j]);
+    if (getDirectionIndex(direction, i, j) === null){
```

Comment:
Think that this condition is being handled by https://github.com/amishas157/reversi-react-electron/pull/1/files#diff-1fdf421c05c1140f6d71444ea2b27638R214

```
 if (i && j && squares[i][j] === player) {
```

If either of `i` or `j` is `null`, `[]` will be returned